services/teams/access_policies/schemas.go

```diff
@@ -53,7 +53,7 @@ var accessPolicySchema = map[string]*schema.Schema{
 	},
 }
 
-var actionRegex = regexp.MustCompile(`^([a-z]*):(\*|[a-zA-Z0-9]+)$`)
+var actionRegex = regexp.MustCompile(`^(\*|[a-z]*):(\*|[a-zA-Z0-9]+)$`)
```

Comment:
@kbudkaLeanspace I just checked the regex in Teams and it is `^([a-zA-Z0-9*]+):([a-zA-Z0-9*]+)$"` which means it's possible to have `test*test:*`. Is it wanted?

Reply:
yes, its ok; this way you can have i.e. `assets:*Node` (everything on nodes) or `*:get*` (full read only access) etc